fhir-persistence-proxy/src/main/java/com/ibm/fhir/persistence/proxy/FHIRProxyConnection.java

```diff
@@ -348,6 +348,7 @@ public boolean isWrapperFor(Class<?> iface) throws SQLException {
 
     @Override
     public void setSchema(String schema) throws SQLException {
+        logger.warning("Calling setSchema, which exposes a transaction bug in Liberty+Derby: " + schema);
```

Comment:
is there a way to scope the warning to just derby dbs?
and maybe reference the issue from a comment?  I think I saw you open one with openliberty but now I can't seem to locate it.

Reply:
I thought it was naturally only used in Derby

Reply:
This will only be hit from Derby when executing unit-tests. It should never be hit when running in Liberty. However, I found a similar (but not the same) defect when executing alter table statements in Derby and I was worried we might somehow be making a call here, hence the log message. If we see this message in our FHIR server logs, it's a bug.